Tutorial.ipynb

```diff
@@ -1,1105 +1,532 @@
-    "\n",
-    "#pre-process input data and make predictions (method 1 - using lists)\n",
-    "titles=df['title'].tolist()\n",
-    "abstracts=df['abstract'].tolist()\n",
-    "sep_token = predictor.preproc.get_tokenizer().sep_token\n",
-    "texts = [\" \".join([title, sep_token, abstract]) for title, abstract in zip(titles, abstracts)] \n",
-    "# Make predictions, checking how long they take to make (note- there are 1042 examples here this is using a (likely NVIDIA P100) GPU)\n",
-    "%time predictions=predictor.predict((texts))\n",
-    "\n",
-    "#pre-process input data and make predictions (method 2 - using dataframes, hard coded SEP token)\n",
-    "#texts=(df['title'].str.strip() + ' [SEP] ' + df['abstract'].str.strip()).to_numpy()\n",
-    "#predictions2=predictor.predict((texts))\n",
-    "\n",
-    "# pre process existing labels \n",
-    "df['class']=df['class'].astype('bool')\n",
-    "y_val=df['class'].to_numpy()"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "14bbc95d",
-   "metadata": {
-    "papermill": {
-     "duration": 0.015171,
-     "end_time": "2022-04-19T18:37:32.649613",
-     "exception": false,
-     "start_time": "2022-04-19T18:37:32.634442",
-     "status": "completed"
-    },
-    "tags": []
-   },
-   "source": [
-    "<h3> Checking performance now </h3>"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 5,
-   "id": "7aa222a1",
-   "metadata": {
-    "execution": {
-     "iopub.execute_input": "2022-04-19T18:37:32.682789Z",
-     "iopub.status.busy": "2022-04-19T18:37:32.681943Z",
-     "iopub.status.idle": "2022-04-19T18:37:32.704675Z",
-     "shell.execute_reply": "2022-04-19T18:37:32.704198Z",
-     "shell.execute_reply.started": "2022-04-19T18:06:29.666948Z"
+      "cell_type": "code",
+      "execution_count": 11,
+      "metadata": {
+        "id": "8781f9db"
+      },
+      "outputs": [],
+      "source": [
+        "!git clone -q https://github.com/PathwayCommons/pathway-abstract-classifier.git"
+      ],
+      "id": "8781f9db"
     },
-    "papermill": {
-     "duration": 0.040272,
-     "end_time": "2022-04-19T18:37:32.704797",
-     "exception": false,
-     "start_time": "2022-04-19T18:37:32.664525",
-     "status": "completed"
+    {
+      "cell_type": "code",
+      "execution_count": 12,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "4sTpfqU138VU",
+        "outputId": "f2473867-0452-4d77-910c-53f3eb9cdf0d"
+      },
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "/content/pathway-abstract-classifier/pathway-abstract-classifier\n",
+            "archive  dev-requirements.txt  requirements.txt  Tutorial.ipynb\n",
+            "Data\t README.md\t       tests\n"
+          ]
+        }
+      ],
+      "source": [
+        "%cd pathway-abstract-classifier\n",
+        "!ls\n",
```

Comment:
```suggestion
```

Looks like this is a debugging statement and was kept by accident

Reply:
I kept it there for some transparency into filesystem the way kaggle does by default - but it's cleaner without and am happy to change if you prefer.